examples/OpticalTracker/scripts/test_energy_deposition.C

```diff
@@ -0,0 +1,22 @@
+#include "CLHEP/Units/SystemOfUnits.h"
```

Comment:
```suggestion
#include "DD4hep/DD4hepUnits.h"
```

Reply:
And dd4hep::ev etc

Reply:
Depending on if gean4 units are used or not one has to convert the energy in the file. Or convert the energy to the expected unit before it is written to file.

Reply:
Actually, here one should just use geant4 units in all cases.